graph/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "sapphire-proxy",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Comment:
Could you also add linters (prettier) and formatters (eslint) as in the `clients/js` repo? I know this is a small proxy, but it helps to have them if this is supposed to be long-lived software. You can move the prettierrc from the clients repo to the root.